index.html

```diff
@@ -0,0 +1,2199 @@
+                  <a href="#">
+                    <img class="w-100  full-height" src="https://d.newsweek.com/en/full/1579893/btn-sports-banner-basketball.jpg" alt="">
+                  </a>
+                </div>
+              </div>
+              <div class="d-flex align-items-center py-2 ">
+                <span class="cover-span p-1">COVER</span>
+                <a href="#" class="mag_link font-weight-bold px-2">NEWS</a>
+              </div>
+              <div>
+                <a href="#">
+                  <h2 class="mag_link2">
+                    Who Should Doctors Save? Inside the Debate Over Rationing Coronavirus Care
+                  </h2>
+                </a>
+              </div>
+            </article>
+            <article class="col-md-3 col-sm-12 pt-5 article2 w-100 " >
+              <div class="w-100">
+                <div class="w-100 ">
+                  <a href="#">
+                    <img class="w-100  full-height" src="https://d.newsweek.com/en/full/1579893/btn-sports-banner-basketball.jpg" alt="">
+                  </a>
+                </div>
+              </div>
+              <div class="d-flex align-items-center py-2 ">
+                <span class="cover-span p-1">COVER</span>
+                <a href="#" class="mag_link font-weight-bold px-2">NEWS</a>
+              </div>
+              <div>
+                <a href="#">
+                  <h2 class="mag_link2">
+                    Who Should Doctors Save? Inside the Debate Over Rationing Coronavirus Care
+                  </h2>
+                </a>
+              </div>
+            </article>
+            <article class="col-md-3 col-sm-12 pt-5 article2 w-100 " >
+              <div class="w-100">
+                <div class="w-100 ">
+                  <a href="#">
+                    <img class="w-100  full-height" src="https://d.newsweek.com/en/full/1579893/btn-sports-banner-basketball.jpg" alt="">
+                  </a>
+                </div>
+              </div>
+              <div class="d-flex align-items-center py-2 ">
+                <span class="cover-span p-1">COVER</span>
+                <a href="#" class="mag_link font-weight-bold px-2">NEWS</a>
+              </div>
+              <div>
+                <a href="#">
+                  <h2 class="mag_link2">
+                    Who Should Doctors Save? Inside the Debate Over Rationing Coronavirus Care
+                  </h2>
+                </a>
+              </div>
+            </article>
+          </div>
+        </section>
+          <div class="editor'spick_container">
+            <section class="in_the_magazine wrapper_1">
+              <div class="d-flex align-items-center justify-content-center border-bottom font-roboto w-100 py-3 mag-heading w-100">
+                <img class="newsweek-icon" src="./new_icon.PNG" alt="newsweek-logo">
+                <h2 class=" mag-title px-2">EDITOR'S PICK</h2>
+              </div>
+                <div class="row">
```

Comment:
The articles in this section have different image sizes and titles. Please make them look like the original page.
Please edit the same on all the screen sizes.
Yours:
![Screenshot from 2020-04-22 13-19-25](https://user-images.githubusercontent.com/51926937/80018706-0f056d80-849c-11ea-838b-d55fd06e1547.png)

Original:
![Screenshot from 2020-04-22 13-19-19](https://user-images.githubusercontent.com/51926937/80018725-1462b800-849c-11ea-9ee8-376412da648c.png)